core/HoistApp.js

```diff
@@ -62,6 +63,21 @@ export function HoistApp(C) {
             };
         },
 
+        /**
+         * App must implement this method with appropriate logic to display a message when the app
+         * has been idle for the amount of minutes defined by the `xhxhIdleTimeoutMins` config.
+         * @param {Function} callback - A callback function to be handled by the showIdleDialog
+         */
+        showIdleDialog(callback) {
```

Comment:
perhaps this should be named 'renderIdleDialog()' if its job is to return a react element that can be rendered.